app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,38 @@
+class TicketsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :read_units, only: :new
+
+  def show
+    @ticket = Ticket.find(params[:id]).decorate
+  end
+
+  def new
+    @ticket = Ticket.new
+    authorize @ticket
+  end
+
+  def create
+    @ticket = Ticket.new(ticket_params)
```

Comment:
Має через користувача робитись
```suggestion
    @ticket = current_user.tickets.build(ticket_params)
```